pkg/service/cluster.go

```diff
@@ -15,20 +15,20 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
-func ListNs(clusterName string) []string {
+func ListNs(clusterName string, ctx context.Context) []string {
```

Comment:
A small suggestion: if we switch to a normal logger instead of `SugaredLogger` please make clusterName as a field to the logger, instead of adding cluster name every log line.